src/installer/pkg/sfx/Microsoft.NETCore.App/monocrossaot.sfxproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.Build.NoTargets" DefaultTargets="BuildMonoCrossAllTargets">
+
+  <PropertyGroup>
+    <MonoAotTargets Condition="'$(MonoAotTargets)' == ''">android-x64;android-arm64;android-x86;android-arm</MonoAotTargets>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <MonoAotTargetRids Include="$(MonoAotTargets.Split(';'))" />
+  </ItemGroup>
+
+  <Target Name="RestoreSubProject">
+    <MSBuild Targets="Restore"
+             Projects="$(MSBuildThisFileDirectory)Microsoft.NETCore.App.MonoCrossAOT.sfxproj" />
+  </Target>
```

Comment:
This causes a separate restore invocation to be triggered which we don't want for different reasons (reliability, performance, etc.). Why is this necessary? If you use the Traversal SDK, you can just add ProjectReference to that project if you don't want to expose it in Subsets.props.

Reply:
The build was failing without the explicit second call to restore. I know it's ugly, but it's the workaround I found which worked.

Your suggestion is adding a ProjectReference to Microsoft.NETCore.App.MonoCrossAOT.sfxproj in monocrossaot.sfxproj, and switching Microsoft.NETCore.App.MonoCrossAOT.sfxproj to Traversal instead of NoTargets?

Reply:
> Your suggestion is adding a ProjectReference to Microsoft.NETCore.App.MonoCrossAOT.sfxproj in monocrossaot.sfxproj, and switching Microsoft.NETCore.App.MonoCrossAOT.sfxproj to Traversal instead of NoTargets?

Right, but only if that project isn't restored yet. What was the failure that you were hitting?